src/redux/paymentDetailsSlice.js

```diff
@@ -0,0 +1,45 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import { getPaymentDetails } from '../api';
+
+const initialState = {
+  paymentDetails: [],
+  paymentDetailsLoading: true,
+  paymentDetailsError: [],
+  activeOrBoundPaymentDetails: []
+};
+
+export const fetchPaymentDetails = createAsyncThunk(
+  'paymentDetails/paymentDetails',
+  async (_, { rejectWithValue }) => {
```

Comment:
I like this solution with `_` for the unused parameter. Could you apply this solution also in `submitPaymentWithoutDetails` and `fetchCustomerOffers`?